go/state/state.go

```diff
@@ -85,6 +88,8 @@ type LiveDB interface {
 	GetBalance(address common.Address) (balance common.Balance, err error)
 	GetNonce(address common.Address) (nonce common.Nonce, err error)
 	GetStorage(address common.Address, key common.Key) (value common.Value, err error)
+	// HasEmptyStorage returns true if account has empty storage.
```

Comment:
nit: I would remove this document line just because nothing else here is documented ...